eval/Containerfile

```diff
@@ -6,7 +6,6 @@ RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc
 
 WORKDIR /app
 
-COPY eval/requirements.txt .
```

Comment:
Please update the `RUN` command below as well. `eval/requirements.txt` doesn't exist anymore.

Can you please apply also the other comments I had here?
https://github.com/redhat-et/ilab-on-ocp/pull/6#discussion_r1750146938


Since you're pulling from your fork it should ideally look like this:

```
RUN python3.11 -m ensurepip \
    && python3.11 -m pip install --no-cache-dir git+https://github.com/sallyom/eval@add-api-key
```

That's it. No repo cloning (the repo ends up in the image with whole git history, that is not necessary at all), no extra `requirements.txt`s.

Reply:
but I'll test it again to be sure it's good w/out the extra requirements.txt

Reply:
all good :) (you already knew that) removed, updated!